src/Symfony/Component/Config/Tests/Builder/Fixtures/ArrayExtraKeys/Symfony/Config/ArrayExtraKeys/BarConfig.php

```diff
@@ -71,9 +71,10 @@ public function toArray(): array
     
     /**
      * @param ParamConfigurator|mixed $value
```

Comment:
should probably be removed, and changed to `mixed` in 5.4

Reply:
Adding ParamConfigurator provides better DX afaik since phpstorm can leverage that to make suggestions.

Reply:
but then, why is it removed in the AddToList fixture in 6.0 if it is useful ?

Side note: I'm not sure passing a ParamConfigurator will be the common case (you are more likely to pass an actual value) so I'm not sure this suggestion is even useful.